docs/release-notes/v12.1.0.md

```diff
@@ -0,0 +1,121 @@
+---
+id: v12.1.0
+title: Version 12.1.0 Release Notes
+sidebar_label: v12.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_January 18th, 2024_
+
+
+:::info
+Mia-Platform Console v12.1.0 is **now in Preview!**
+
+Console SaaS users can try out v12.1.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+### New Features
+
+#### New Console backoffice configuration using Mia-Platform Microfrontend Composer
+
+Console CMS has been replaced with a new backoffice adopting [Mia-Platform Microfrontend Composer](/microfrontend-composer/what-is.md). 
+At runtime, there will be micro-lc that replaces the previously adopted cms-backend, cms-site and v1-adapter.
+
+For self-hosted installations, take a look at the information given on the [How to upgrade Console guide](/infrastructure/self-hosted/installation-chart/100-how-to-upgrade.md#v12---version-upgrades).
+
+#### New User events for the Webhook section
+
+User events have been added that allow a Company Owner to configure and trigger specific webhooks when changes to user access management occur.
+
+Listed below are the events that have been added with this release:
+
+* User added to Company
+* User Company role edited.
+* User removed from Company
+* User added to Project
+* User removed from Project
+
+For more information regarding individual events, please [go to the documentation](/development_suite/webhooks-and-events/events.mdx#user-added-to-company).
+
+#### Complete Company access management through `miactl`
+
+The new version of `miactl` allows complete management of all identities access to the Company.
+More specifically, starting from today it will be possible to add, modify and remove any type of identity (User, Service Account, Group) simply through command line.
+
+For more information on the specific commands introduced, please [go to the documentation](/cli/miactl/30_commands.md).
```

Comment:
What do you think about:

```suggestion

### New Console backoffice configuration using Mia-Platform Microfrontend Composer

Console CMS has been replaced with a new backoffice adopting [Mia-Platform Microfrontend Composer](/microfrontend-composer/what-is.md). 
At runtime, there will be micro-lc that replaces the previously adopted cms-backend, cms-site and v1-adapter.

For self-hosted installations, take a look at the information given on the [How to upgrade Console guide](/infrastructure/self-hosted/installation-chart/100-how-to-upgrade.md#v12---version-upgrades).

### New User events for the Webhook section

User events have been added that allow a Company Owner to configure and trigger specific webhooks when changes to user access management occur.

Listed below are the events that have been added with this release:

* User added to Company
* User Company role edited.
* User removed from Company
* User added to Project
* User removed from Project

For more information regarding individual events, please [go to the documentation](/development_suite/webhooks-and-events/events.mdx#user-added-to-company).

### Complete Company access management through `miactl`

The new version of `miactl` allows complete management of all identities access to the Company.
More specifically, starting from today it will be possible to add, modify and remove any type of identity (User, Service Account, Group) simply through command line.

For more information on the specific commands introduced, please [go to the documentation](/cli/miactl/30_commands.md).
```